setup.cfg

```diff
@@ -65,3 +65,6 @@ ignore_errors = False
 
 [mypy-bokeh.util.paths]
 ignore_errors = False
+
+[mypy-bokeh.util.options]
+ignore_errors = False
```

Comment:
@mattpap @enricapq is there way to condense these into one section under a single mypy config block? Guessing not, but it's going to result in a huge even messier list in this file before we get to the end.

Reply:
@bryevdv, I will conflate those as soon as we have enough coverage. For now this results in simpler merge conflicts to resolve.